setup.cfg

```
@@ -29,7 +29,7 @@ python_requires = >=3.7
 install_requires =
     inflection
     factory_boy>=2.10.0
-    pytest>=4.6
+    pytest>=6.0
```

pytest 6.2.4 is the first to work with Python 3.10, and 6.2.5 the first to officially support 3.10:

https://docs.pytest.org/en/7.1.x/changelog.html#pytest-6-2-5-2021-08-29

```suggestion
    pytest>=6.2.4
```

(`tox.ini` can be updated to match.)

It's possible to have conditional installs for different versions, but it would greatly simplify testing/maintenance to require the same version for all.